R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -26,7 +26,7 @@ sc <- sparkR.init()
 sqlContext <- sparkRSQL.init(sc)
 
 test_that("glm and predict", {
-  training <- createDataFrame(sqlContext, iris)
+  training <- suppressWarnings(createDataFrame(sqlContext, iris))
```

Comment:
could you add a note in JIRA SPARK-11976 that once that is fixed we should remove `suppressWarnings` around DF from iris?